src/pages/en/guides/images.md

```diff
@@ -10,11 +10,12 @@ Astro provides several ways for you to use images on your site, whether they are
 
 ### In `.astro` files
 
-Astro uses standard HTML `<img>` or `<img />` elements to display images within your `.astro` files. All HTML image attributes are supported.
+Astro uses standard HTML `<img>` or `<img />` elements to display images, and all HTML image attributes are supported. 
```

Comment:
I read “`<img>` or `<img />`” as a typo at first — like trying to work out what the second was supposed to be instead. Do you think we can get away without showing the self-closing version?

```suggestion
Astro uses standard HTML `<img>` elements to display images, and all HTML image attributes are supported. 
```

Reply:
Or — if we’re really concerned that `<img>` doesn’t work in MDX — we could write:

```suggestion
Astro uses standard HTML `<img />` elements to display images, and all HTML image attributes are supported. 
```

Although I think this might be technically inaccurate in that the “standard” HTML element is specifically `<img>` without the closing slash. 🤷

Reply:
I think we only really need to worry about what works in Astro (`.astro`) files here in this section. And, I think both work? Do you think it's fine just to say "HTML `<img>`" (we drop the "standard" and the self-closing one)?  Wait until someone complains?

Reply:
“HTML `<img>`” works! And both work in `.astro` yup.